tests/integration_tests/test_integration_mssql_example.py

```diff
@@ -0,0 +1,73 @@
+import logging
+from typing import Generator
+
+import pytest
+import sqlalchemy as sqlalchemy
+from sqlalchemy import func, select, table
+
+from fidesops.db.session import get_db_session, get_db_engine
+
+logger = logging.getLogger(__name__)
+
+
+MSSQL_URL_TEMPLATE = "mssql+pyodbc://sa:Mssql_pw1@mssql_example:1433/{}?driver=ODBC+Driver+17+for+SQL+Server"
+MSSQL_URL = MSSQL_URL_TEMPLATE.format("mssql_example")
+MASTER_MSSQL_URL = MSSQL_URL_TEMPLATE.format("master") + "&autocommit=True"
+
+
+@pytest.fixture(scope="module")
+def mssql_setup():
+    """
+    Set up the SQL Server Database for testing.
+    The query file must have each query on a separate line.
+    Initial connection must be done to the master database.
+    """
+    engine = sqlalchemy.create_engine(MASTER_MSSQL_URL)
+    with open("data/sql/mssql_example.sql", "r") as query_file:
+        queries = [query for query in query_file.read().splitlines() if query != ""]
+    print(queries)
+    for query in queries:
+        engine.execute(sqlalchemy.sql.text(query))
+    yield engine
+    engine.dispose()
```

Comment:
it's a tough one since scoping is different for each fixture. I'll see if I can find a good workaround, maybe dynamic scoping.

Reply:
we shouldn't need this either if we can figure out to run the mssql_example.sql file as part of bringing up the mssql docker container 🤞

Reply:
true! After speaking with @ThomasLaPiana, it didn't seem possible to run the `mssql_example` SQL file using `volumes` in our `docker-compose.integration-test.yml` file. This is specific to mssql, even though it is possible in mysql. 

That's why he suggested the pythonic way of doing it via fixtures. I wonder if there's a way we can trigger a python module to run after docker container is brought up.

Reply:
yes mssql docker has some frustrating points - i linked to some possible workarounds in the hidden comment below https://github.com/ethyca/fidesops/pull/151#discussion_r780434222

While it might take a little while to get this piece working, I think the investment is important so we can execute privacy requests in docker against mssql outside of tests.

Reply:
trying this: https://docs.docker.com/engine/reference/commandline/exec/#extended-description, so I can run .py files specifically on the mssql_example container